src/brsTypes/BrsType.ts

```diff
@@ -132,6 +132,10 @@ export function getBrsValueFromFieldType(type: string, value?: string): BrsType
         case "string":
             returnValue = value ? new BrsString(value) : new BrsString("");
             break;
+        case "uri":
```

Comment:
💭 if we leave this as a `string`, should we just make pull this up as an additional case statement alongside `"string"`, since the logic is equivalent?

Reply:
Ah, yes, good call!